src/Symfony/Component/HttpFoundation/Tests/JsonResponseTest.php

```diff
@@ -224,7 +224,7 @@ class JsonSerializableObject implements \JsonSerializable
     {
         public function jsonSerialize()
         {
-            trigger_error('This error is expected', E_USER_WARNING);
+            @trigger_error('This error is expected', E_USER_WARNING);
```

Comment:
This one is not a deprecation, please revert and ensure that you silence only deprecations